ghscrap/scrapper.py

```diff
@@ -113,7 +100,13 @@ def _search_location(soup, max_users, params):
 
     users = []
     for i in range(1, number_of_pages):
-        users.extend(search_page_users(i, params=params))
+        parsed_users=search_page_users(i, params=params)
```

Comment:
Its a valid solution to the problem.
However, I think this complicates the process too much.

Adding a single check to length of list `users` should be enough !

For example:
If the user wants 15 names, we can scrape 20 and return the first 15.
Thats fine by me.